changelog.txt

```diff
@@ -1,5 +1,8 @@
 *** Changelog ***
 
+2020.04.16 - version 2.0.4
```

Comment:
Shouldn't this be today's date?

Reply:
Oops! Fixed directly in the original commit =)